tests/framework/extensions/clusters/clusters.go

```diff
@@ -33,6 +34,22 @@ const (
 	FleetSteveResourceType        = "fleet.cattle.io.cluster"
 )
 
+// GetV1ProvisioningClusterByName is a helper function that returns the cluster ID by name
```

Comment:
I will approve but we should consider making a tech debt item for both `GetV1ProvisioningClusterByName` and `GetClusterIDByName` to be able to get the corresponding objects without having to loop through each one.

Reply:
For sure. I'll make the ticket, we can definitely leverage the list options to do so.